make/addons.mk

```
@@ -84,3 +84,11 @@ generate-mindthegap-repofile: generate-helm-configmap ; $(info $(M) generating h
 .PHONY: template-helm-repository
 template-helm-repository: generate-mindthegap-repofile ## this is used by gorealeaser to set the helm value to this.
 	sed -i '/RepositoryURL:/s#\(RepositoryURL: *\)\(.*\)#\1{{ if .Values.selfHostedRegistry }}oci://helm-repository.{{ .Release.Namespace }}.svc/charts{{ else }}\2{{ end }}#'  "./charts/cluster-api-runtime-extensions-nutanix/templates/helm-config.yaml"
+
+.PHONY: list-images
+list-images:
+	cd hack/tools/fetch-images && go build
+	./hack/tools/fetch-images/fetch-images \
```

Dropping the redirect would allow for this:
```suggestion
	cd hack/tools/fetch-images && run . \
```